AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPy/QtPyAssetEditor.py

```diff
@@ -141,3 +224,52 @@ def expand_qt_container_rows(self, category_name: str) -> None:
         returns: None
         """
         super().expand_qt_container_rows(self.asset_editor_row_container, category_name)
+
+    def refresh_qt_references(self):
+        """
+        Function for updating references to qt objects associated with the asset manager. Expanding/deleting stuff
+        from the list will cause the hierarchy to shift around so references can get stale, which makes interacting
+        with it unpredictable
+
+        returns: None
+        """
+        self.asset_editor_widget = self.asset_editor.findChild(QtWidgets.QWidget, "AssetEditorWindowClass")
+        helper.wait_for_condition(lambda: self.asset_editor_widget is not None, WAIT_TIME_SEC_3)
+        self.asset_editor_row_container = self.asset_editor_widget.findChild(QtWidgets.QWidget, "ContainerForRows")
+        helper.wait_for_condition(lambda: self.asset_editor_row_container is not None, WAIT_TIME_SEC_3)
```

Comment:
This function only needs calling once, after the new asset has been created. Before that, the editor widget and row container do not exist. They will not change once created.

Reply:
removed extra calls to this function. handles are only refreshed now when the event and parameter are created.